download/releaseNotes/releaseNotes8.adoc

```diff
@@ -32,6 +32,19 @@ You can now modify your streams using link:https://docs.optaplanner.org/latestFi
 We have made some tweaks under the hood
 so that your experience with the recently released link:https://openjdk.java.net/projects/jdk/16/[OpenJDK 16] continues to be smooth.
 
+=== Inject and Autowire ConstraintVerifier in Quarkus and Spring Boot
+
+You can now link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsTestingQuarkus[inject the Constraint Verifier in Quarkus] and
+link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsTestingSpringBoot[autowire the Constraint Verifier in Spring Boot], allowing
+you to easily test your link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreams[ConstraintProvider constraints].
```

Comment:
```suggestion
you to link:https://docs.optaplanner.org/latestFinal/optaplanner-docs/html_single/index.html#constraintStreamsTesting[test your constraints] more easily.
```

Reply:
I don't agree with "test your constraints" since that implies it can be used with DRL's, EasyScoreCalculator, IncrementalScoreCalculator, etc. but can only be use with ConstraintStreams/ConstraintProvider

Reply:
Fair point. "test your constraint streams works as well.